tests/gradient_descent_test.py

```diff
@@ -25,6 +25,8 @@
 from sklearn import datasets
 
 
+n_calls = 0
```

Comment:
use uppercase for global constants, e.g., N_CALLS=0

Reply:
done